src/vnsw/agent/oper/interface.cc

```diff
@@ -451,7 +453,7 @@ void Interface::GetOsParams(Agent *agent) {
             "> querying mac-address for interface <" << name << "> " <<
             "Agent-index <" << id_ << ">");
         os_oper_state_ = false;
-        close(fd);
+        closesocket(fd);
```

Comment:
As per our Tuesday discussion, we should avoid breaking compatibility if possible.